modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -0,0 +1,114 @@
+//
+// * rosa_backing_up_and_restoring_applications/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-installing-oadp-rosa-sts_{context}"]
+= Intsalling OADP on ROSA with STS
+
+AWS STS is a global web service that provides short-term credentials for IAM or federated users. ROSA with STS is the recommended credential mode for ROSA clusters.
+
+[IMPORTANT]
+====
+Disable Restic for OADP in ROSA environments because Restic is not supported in such environments. Instead use the native snapshots workflow to backup volumes.
+====
+
+.Prerequisites
+
+* A ROSA OpenShift Cluster with the required access and tokens.
+* link:https://docs.openshift.com/container-platform/4.12/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-creating-default-secret_installing-oadp-aws[A default Secret], if your backup and snapshot locations use the same credentials, or if you do not require a snapshot location.
+* An OpenShift secret that has been created from your AWS token file.
++
+[source,yaml]
+----
+cat <<EOF > ${SCRATCH}/credentials
+[default]
+role_arn = ${ROLE_ARN}
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+EOF
+oc -n openshift-adp create secret generic cloud-credentials \
+  --from-file=${SCRATCH}/credentials
+----
+
+.Procedure
+
+. Install the OADP Operator.
+. Create AWS cloud storage using your AWS credentials:
++
+[source,yaml]
+----
+cat << EOF | oc create -f -
+apiVersion: oadp.openshift.io/v1alpha1
+kind: CloudStorage
+metadata:
+  name: ${CLUSTER_NAME}-oadp
+  namespace: openshift-adp
+spec:
+  creationSecret:
+    key: credentials
+    name: cloud-credentials
+  enableSharedConfig: true
+  name: ${CLUSTER_NAME}-oadp
+  provider: aws
+  region: $REGION
+EOF
+----
+. Create the `DataProtectionApplication resource`, which is used to configure the connection to the storage where the backups and volume snapshots will be stored:
++
+[NOTE]
+====
+The `enable` parameter of `restic` is set to `false` in this configuration because OADP does not support Restic in ROSA environments.
+====
++
+[source,yaml]
+----
+cat << EOF | oc create -f -
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: ${CLUSTER_NAME}-dpa
+  namespace: openshift-adp
+spec:
+  backupLocations:
+  - bucket:
+      cloudStorageRef:
+        name: ${CLUSTER_NAME}-oadp
+      credential:
+        key: credentials
+        name: cloud-credentials
+      default: true
+  configuration:
+    velero:
+      defaultPlugins:
+      - openshift
+      - aws
+      restic:
+        enable: false
+  volumeSnapshots:
+  - velero:
+      config:
+        credentialsFile: /tmp/credentials/openshift-adp/cloud-credentials-credentials
+        enableSharedConfig: "true"
+        region: ${REGION}
+      provider: aws
+EOF
+----
++
+You are now ready to backup and restore OpenShift applications, please refer to the OADP documentation for instructions.
```

Comment:
This is an opportunity to say "See _Additional resources_ for more information", and include the link there.

Reply:
Good idea -- I executed it a bit differently, following the source material for this PR: https://hackmd.io/yiZBeGFkSJOGIwni9iNnsA. Kathryn told me to use a hardlink to the openshift site for this kind of reference.

Reply:
Gotcha, I was unaware there are specific rules in regards to ROSA. We should update the peer review checklist to include that detail.

Reply:
Happy to help! Thanks for your consideration and changes!